### PR TITLE
[INJICERT-41] new issuer for inji certify testing

### PR DESCRIPTION
**Pull request by challabeehyv**

### CODE REVIEW

File: mimoto-issuers-config.json [hunk 1]
```diff
@@ -408,6 +408,43 @@
...
       "credential_audience": "https://${mosip.esignet.insurance.host}",
       "enabled": "true"
+    },
+    {
+      "credential_issuer": "VeridoniaCertify",
+      "display": [
+        {
+          "name": "Veridonia Insurance Company (Certify)",
+          "logo": {
+            "url": "https://sunbird.org/images/sunbird-logo-new.png",
```
Comment (vishwa-vyom) [inline mimoto-issuers-config.json]: To be consistent, can we change this to https://${mosip.api.public.host}/inji/veridonia-logo.png

File: mimoto-issuers-config.json [hunk 2]
```diff
@@ -408,6 +408,43 @@
...
+      "additional_headers": {
+        "Accept": "application/json"
+      },
+      ".well-known": "https://${mosip.injicertify.host}/v1/certify/issuance/.well-known/openid-credential-issuer?version=v11",
+      "redirect_uri": "io.mosip.residentapp.inji://oauthredirect",
+      "authorization_endpoint": "https://${mosip.esignet.insurance.host}/authorize",
+      "authorization_audience": "https://${mosip.esignet.insurance.host}/v1/esignet/oauth/v2/token",
+      "token_endpoint": "https://${mosip.api.public.host}/residentmobileapp/get-token/VeridoniaCertify",
+      "proxy_token_endpoint": "https://${mosip.esignet.insurance.host}/v1/esignet/oauth/v2/token",
+      "credential_endpoint": "https://${mosip.esignet.insurance.host}/v1/esignet/vci/credential",
```
Comment (vishwa-vyom) [inline mimoto-issuers-config.json]: this is misleading, possible to keep this empty or give out vci url ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: mimoto-issuers-config.json (hunk 1 above)
Comment (challabeehyv) [reply mimoto-issuers-config.json]: changed this

File: mimoto-issuers-config.json (hunk 2 above)
Comment (challabeehyv) [reply mimoto-issuers-config.json]: changed to empty

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**